geonode/facets/tests.py

```diff
@@ -525,6 +550,13 @@ def t(tk):
             (regname, {t1flt: [t("1_1"), t("1_0")]}, {"R0": 2, "R1": 3, "R2": 4}),
             (regname, {t1flt: t("1_1"), "key": ["R0", "R1"]}, {"R1": 1, "R0": None}),
             (regname, {t1flt: t("1_1"), "key": ["R0"]}, {"R0": None}),
+            # groups
+            (grname, {grflt: [4, 5], "key": [4, 5]}, {4: 7, 5: 3}),
+            (grname, {grflt: [4], "key": [4]}, {4: 7}),
```

Comment:
Are we using explicit group IDs? Where do they come from?
Pls note that for all other objects we store the IDs generated by the DB in some sort of dict (`thesauri_k`) or use some slug (`R0`, `R1`,`R2` for regions, `C0`, `C1`, `C2` for categories)...